lib/datadog/grok/src/grok_filter.rs

```diff
@@ -0,0 +1,153 @@
+use crate::ast::{Function, FunctionArgument};
+use crate::parse_grok::Error as GrokRuntimeError;
+use crate::parse_grok_rules::Error as GrokStaticError;
+use ordered_float::NotNan;
+use std::{convert::TryFrom, string::ToString};
+use strum_macros::Display;
+use vrl_compiler::Value;
+
+#[derive(Debug, Display, Clone)]
+pub enum GrokFilter {
+    Integer,
+    IntegerExt,
```

Comment:
I think this could do with a comment -- it's not immediately obvious how `IntegerExt` and `NumberExt` differ from the base equivalents.